src/style.css

```
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --link-color: #00acdc;
+  --header-links-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  line-height: 1px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link:link {
+  display: block;
+  text-align: center;
+  color: black;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  height: var(--header-links-height);
+  line-height: var(--header-links-height);
+}
+
+.nav__link:visited {
+  color: black;
+}
+
+.nav__link:hover {
+  color: var(--link-color);
+}
+
+.nav__link.is-active {
+  color: var(--link-color);
+}
+
+.is-active::after {
+  position: relative;
+  top: -4px;
```

```suggestion
  position: absolute;
  bottom: 0;
```
You can add `position: relative;` to the link and use `absolute` here, in this case you don't need to calculate pixels